sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Added "transparent" version of `Azure::Core::Uuid` to enable reading or writing from an existing Uuid.
```

Comment:
I'd put it more straighforward as something like "added Uuid::CreateFromArray() and Uuid::AsArray()", but up to you.